apps/web/src/features/entity/composed/ListEntity.css

```
@@ -28,3 +31,69 @@
+
+   The family class goes on whatever needs the geometry — each row, and the
+   soup group headers, which are siblings of the rows inside the virtualizer and
+   so can't read a row's own variables. Soup's per-view registry names the
+   family and hands it to the header; the `row` family splits into wide and
+   narrow by container width, which rows and headers take from the same
+   `isWide` signal. */
+.soup-row-wide,
+.soup-row-narrow,
+.soup-row-card {
+  --soup-row-content-inset: calc(
+    var(--soup-row-gutter) + var(--soup-row-padding-l) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)